gitvs/src/Components/ButtonAppearProfile.js

```diff
@@ -0,0 +1,5 @@
+function ButtonAppearProfile(id) {
+  document.getElementById(id).style.display = 'block'
```

Comment:
You do not need vanilla JS to control style, use state instead

Reply:
I made the changes